modules/apps/frontend-js/frontend-js-web/src/main/resources/META-INF/resources/liferay/util/show_caps_lock.js

```diff
@@ -0,0 +1,44 @@
+/**
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * This library is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU Lesser General Public License as published by the Free
+ * Software Foundation; either version 2.1 of the License, or (at your option)
+ * any later version.
+ *
+ * This library is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more
+ * details.
+ */
+
+const A_KEY_CODE = 65;
+const F11_KEY_CODE = 122;
+const NUMPAD_1_KEY_CODE = 97;
+const SHIFT_KEY_CODE = 16;
+const Z_KEY_CODE = 90;
+
+export default function showCapsLock(event, spanId) {
+	const span = document.getElementById(spanId);
+
+	if (!span) {
+		throw new TypeError(
+			"the provided spanId doesn't return a valid element"
+		);
+	}
+
+	const {keyCode} = event;
```

Comment:
Hey @kresimir-coko, I think we should get rid of using `keyCode` for future reliable code. See https://developer.mozilla.org/en-US/docs/Web/API/KeyboardEvent/keyCode

> Deprecated: This feature is no longer recommended. Though some browsers might still support it, it may have already been removed from the relevant web standards, may be in the process of being dropped, or may only be kept for compatibility purposes. Avoid using it, and update existing code if possible; see the [compatibility table](https://developer.mozilla.org/en-US/docs/Web/API/KeyboardEvent/keyCode#browser_compatibility) at the bottom of this page to guide your decision. Be aware that this feature may cease to work at any time.

I would recommend you go with event.key instead. Note this feature is not supported on IE11. If this code should be backportable, I think we can create an utility to keyboard events to support the legacy and depreacted way and the future :)

Reply:
Ah, that's relatively recent then, thanks for the catch, gonna change it to `code`